src/components/Footer/footer.ts

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+import colors from "../../core/constants/colors";
+
+const FooterWrapper = styled.footer`
+    margin: auto;
+    bottom: 0;
+    width: 100%;
+    background-color:  ${colors.WHITE_COLOR};    
+    border-top: 1px solid ${colors.LIGHT_GRAY_COLOR};
+    padding: 20px 0 20px 0;
```

Comment:
padding-top:20px; padding-bottom:20px ?

Reply:
Resolved in 642a3efafd4530938de95580638849f0b1cbf8dc